README.md

```diff
@@ -1,3 +1,53 @@
 # plugin-api
 
-tbd
+This repository consisted of APIs used in gotify plugins.
+
+## Usage
+
+### Plugin API
+
+See https://github.com/gotify/server/issues/51
+
+TODO: docs
+
+### CLI tools
+
+This repository contained two CLI utils in order to resolve compatibility issues cause by common import paths and symbol sharing.
+
+#### `github.com/gotify/cmd/gomod-cap`
+
+Since `go.mod` follows a [minimal version selection](https://github.com/golang/proposal/blob/master/design/24301-versioned-go.md), packages are built with the lowest common version requirement defined in `go.mod`. This poses a problem when developing plugins:
+
+If the gotify server is built with the following go.mod:
+```
+require some/package v0.1.0
+```
+But when the plugin is built, it used a newer version of this package:
+```
+require some/package v0.1.1
+```
+Since the server is built with `v0.1.0` and the plugin is built with `v0.1.1` of `some/package`, the built plugin could not be loaded due to different import package versions.
+
+`gomod-cap` is a simple util to ensure that plugin `go.mod` files does not have higher version requirements than the main gotify `go.mod` file.
+
+To resolve all incompatible requirements:
+```bash
+$ go run github.com/gotify/cmd/gomod-cap -from /path/to/gotify/server/go.mod -to /path/to/plugin/go.mod
```

Comment:
This should be `github.com/gotify/plugin-api/cmd/gomod-cap` (this typo is also present on line 39)